rt_gene/src/rt_gene/tracker_generic.py

```diff
@@ -101,10 +109,51 @@ def get_distance_matrix(self, new_elements):
                 distance_matrix[i][j] = self._tracked_elements[element_id].compute_distance(new_element)
         return distance_matrix, map_index_to_id
 
+    @staticmethod
+    def align_face_to_eyes(face_img, right_eye_center, left_eye_center, face_width=None, face_height=None):
+        # modified lightly from https://www.pyimagesearch.com/2017/05/22/face-alignment-with-opencv-and-python/
+        desired_left_eye = (0.35, 0.35)
+        desired_face_width = face_height if face_width is not None else face_img.shape[1]
```

Comment:
I am assuming this should be `face_width` rather than `face_height`?

Reply:
you know the answer to this....